scripts/handle-user-data

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-LABEL=boot2docker-data
+LABEL=b2d-data
```

Comment:
Why is this changed?

Reply:
Because xfs label shouldn't more than 11 char, that's the reason I had to change it.

Reply:
Hmmm interesting

Reply:
disk labels are anyways an overrated feature... often a random string of hex characters will lead to better results for identification. although they are often used by humans for reading?! Ugh...